ios-continuous-integration_5c8a397c2c7d3a154460db00.md

```diff
@@ -123,228 +122,218 @@ blocks:
-commands to execute. Blocks, tasks and jobs are Semaphore's [core
-concepts][concepts].
+Each block has a task that defines one or more parallel jobs. Jobs define
+commands that should be executed by the pipeline.
+
+Blocks, tasks and jobs are Semaphore's [core concepts][concepts].
 
 ``` yaml
 blocks:
   - name: Run tests
     task:
+      # This environment variable is exported in every job within this block:
       env_vars:
-        # This environment variable is exported in every job within this block:
         - name: LANG
           value: en_US.UTF-8
       prologue:
         commands:
-          - # this runs before every parallel job
+          - # Commands to run before *every* parallel job
       jobs:
-        - name: Fastlane test
-        commands:
-          - # ...
-        - name: 2nd parallel job
-        commands:
-          - # ...
-        - name: 3rd parallel job
-        commands:
-          - # ...
-  - name: Build app
-    task:
-      jobs:
-        - name: Fastlane app
-        commands:
-          - # ...
+        - name: Run tests
+          commands:
+            - bundle exec xcversion select 11.1
+            - bundle exec fastlane test
+        - name: Second parallel job
+          commands:
+            - # Commands for the second parallel job
+            - # ...
+        - name: Third parallel job
+          commands:
+            - # Commands for the third parallel job
+            - # ...
 ```
 
 ### Downloading code
 
-To download your code in a job, use the [`checkout` command][checkout], an [open
-source script][checkout-source] provided by Semaphore.
+To download your code in a job, use [`checkout`][checkout], an
+[open source script][checkout-source] provided by Semaphore.
 
-By default this performs a shallow git clone from remote origin. If you want a
-full clone which will be cached by Semaphore, use `checkout --use-cache`.
+By default `checkout` performs a shallow git clone from a remote origin. If you
+need a full clone that will be cached by Semaphore, use `checkout --use-cache`.
 
 ``` yaml
-        commands:
-          - checkout
+commands:
+  - checkout
 ```
 
 ### Installing dependencies
 
-Semaphore can install dependencies from any supported language.
-Using the versatile [cache tool][cache], you can save time by reusing them
-across blocks and workflows.
+Semaphore can install dependencies from *any* supported language. Using the
+versatile [`cache`][cache-command] command, you can save time by reusing the
+same dependencies across multiple blocks and workflows.
 
 ``` yaml
-        commands:
-          - cache restore
-          - bundle install --path vendor/bundle
-          - cache store
+commands:
+  - cache restore
+  - bundle install --path vendor/bundle
+  - cache store
 ```
 
-In example configuration, the first command restores dependencies from cache
-using partial cache key matching:
+In the example configuration, the first command restores dependencies from
```

Comment:
We should remove this whole section IMO (until "The cache restore command..."). It's a leftover from the days when `cache (re)store` wasn't able to auto-detect the package manager and dependencies being used, and needed to be given specific cache keys.